untrusted/quote_verify/src/lib.rs

```diff
@@ -185,14 +244,19 @@ impl Quote {
 pub enum Error {
```

Comment:
Optional: Add `derive(displaydoc::Display)`

Reply:
Added `Display`.  I had to convert a multi-line comment into a block comment, not sure if there are any thoughts on mixing these in the same type.

> Multi-line comments are disabled by default by displaydoc. Please consider using block doc comments (/** */)

The error message is a bit misleading in that multi-line comments aren't supported, the subsequent lines are ignored, https://github.com/yaahc/displaydoc#details